sourcegraph/manifest.json

```diff
@@ -0,0 +1,35 @@
+{
+  "manifest_version": "2.0.0",
+  "app_uuid": "19916057-6b27-4c17-89c5-821794532315",
+  "app_id": "sourcegraph",
```

Comment:
Hi @Joelkw 👋 , I'm continuing your Sourcegraph PR in my new PR so I can fix the validation errors faster. In your original PR, there are some issues with the manifest.json file where it contained both v1 and v2 manifest elements. 

I've resolved most of those issues already in this PR, but one of these elements is using `integration_id` (v1) and `app_id` (v2). In your PR, `integration_id` is [`sourcegraph-services-map`](https://github.com/DataDog/integrations-extras/pull/1140/files#diff-058463be0103b47b67a406e94abc1d1b550925d9113ced29d0935460f9391692R2) but your `app_id` is [`sourcegraph`](https://github.com/DataDog/integrations-extras/pull/1140/files#diff-058463be0103b47b67a406e94abc1d1b550925d9113ced29d0935460f9391692R15), although they should be the same. Is there a preferred value to use?

Reply:
Bumping question for @Joelkw

Reply:
@fanny-jiang @yzhan289 thank you for checking! 

Rather than answer that specific question, I wanted to let you know that we're [changing how our extensions APIs work](https://docs.sourcegraph.com/extensions/deprecation), and migrating existing extensions (like this one) over to a new future format. In the meantime, however, this extension is now deprecated, so it probably doesn't make sense to still list this. Instead, we can just close this PR. Thank you all for your help so far. 

cc @ryankscott who leads that team now!

Reply:
@Joelkw Thank you for the update! I'll go ahead and close this PR. Feel free to open a new PR if you wish to add an integration for the new extension API!